RUFAS/biophysical/animal/pen.py

```diff
@@ -596,6 +604,92 @@ def get_manure_data(self) -> PenManureData:
             num_lactating_cows=self.number_of_lactating_cows_in_pen,
             num_stalls=self.num_stalls,
         )
+        manure_streams: list[dict[str, ManureStream]] = self.get_manure_streams()
+
+        return {"pen_manure_data": pen_manure, "manure_streams": manure_streams}
+
+    def get_manure_streams(self) -> list[dict[str, ManureStream]]:
+        """
+        Packages ManureStream objects based on pen AnimalManureExcretions data and input-defined splitting.
+
+        Returns
+        -------
+        list[dict[str, ManureStream]]
+            A list of dictionaries containing identifying stream information
+            and the corresponding ManureStream object.
+        """
+        animal_manure_streams: list[dict[str, ManureStream]] = []
+
+        total_pen_manure_data = NewPenManureData(
+            num_animals=len(self.animals_in_pen),
+            manure_deposition_surface_area=1.0,
+            animal_combination=self.animal_combination,
+            pen_type=self.pen_type,
+            manure_urine_mass=self.total_manure_excretion.urine,
+            manure_urine_nitrogen=self.total_manure_excretion.urine_nitrogen,
+            stream_type=StreamType.GENERAL,
+        )
+
+        total_stream = ManureStream(
+            water=self.total_manure_excretion.manure_mass - self.total_manure_excretion.total_solids,
+            ammoniacal_nitrogen=self.total_manure_excretion.manure_total_ammoniacal_nitrogen,
+            nitrogen=self.total_manure_excretion.manure_nitrogen,
+            phosphorus=self.total_manure_excretion.phosphorus,
+            potassium=self.total_manure_excretion.potassium,
+            ash=0,
+            non_degradable_volatile_solids=self.total_manure_excretion.non_degradable_volatile_solids,
+            degradable_volatile_solids=self.total_manure_excretion.degradable_volatile_solids,
+            total_solids=self.total_manure_excretion.total_solids,
+            volume=self.total_manure_excretion.manure_mass / ManureConstants.MANURE_DENSITY,
+            pen_manure_data=total_pen_manure_data,
+        )
+
+        if self.animal_combination == AnimalCombination.LAC_COW:
+            parlor_stream_proportion = self.minutes_away_for_milking / 1440
+            general_stream_proportion = 1 - parlor_stream_proportion
+            manure_stream = total_stream.split_stream(
```

Comment:
```suggestion
            parlor_stream = total_stream.split_stream(
```
It might be clearer to name something like this, indicating it's a parlor-related stream. Just a suggestion, feel free to have different names.